modules/xmpp/Caps.js

```diff
@@ -133,6 +151,28 @@ export default class Caps extends Listenable {
         this.rooms.forEach(room => room.sendPresence());
     }
 
+    /**
+     * Updates the presences in the rooms based on the current values in externalFeatures.
+     * @param {ChatRoom} room the room.
+     * @private
+     */
+    _updateRoomsWithExternalFeatures(room) {
```

Comment:
This updates a single room, the name and doc should reflect it.

Reply:
I had the same comment. I'm confused by why we say rooms when we really mean one.